pyvrp/cpp/CostEvaluator.h

```diff
@@ -4,8 +4,24 @@
 #include "Measure.h"
 #include "Solution.h"
 
+#include <concepts>
+#include <limits>
+
 namespace pyvrp
 {
+// The following methods must be implemented for a type to be evaluatable by
+// the CostEvaluator.
+template <typename T> concept Evaluatable = requires(T arg)
```

Comment:
Not very strong on this but just wondering if we should we make the name a bit more specific? Since e.g. a move is also 'evaluatable' (it even has a function 'evaluate') but that's something different? Not sure though what the name should be. Maybe `CostEvaluatable` aligns better as subject for `CostEvaluator` but doesn't completely resolve ambiguity with moves...

Reply:
There's no ambiguity if we move this into `CostEvaluator` since that clearly specifies the scope, which I'd prefer over a longer name.

Reply:
Ah that's not allowed as concepts are not allowed in class scope. Oh well. `CostEvaluatable` it is.